bin/crew

```diff
@@ -1690,8 +1713,14 @@ def remove(pkgName)
           puts "Removing #{line} will break crew. It was #{'NOT'.lightred} deleted." if @opt_verbose
         else
           puts "Removing file #{line}".lightred if @opt_verbose
-          puts "fileist contains #{line}".lightred if @opt_verbose && !line.include?(CREW_PREFIX)
-          FileUtils.rm_rf line if line.include?(CREW_PREFIX)
+          puts "filelist contains #{line}".lightred if @opt_verbose && !line.include?(CREW_PREFIX)
+          if line.include?(CREW_PREFIX)
+            if conflicts.include?(line)
+              puts "#{line} is in another package. It will not be removed during the removal of #{pkgName}".orange
+            else
+              FileUtils.rm_rf line
+            end
+          end
```

Comment:
Would `file` be more descriptive of the variable than `line`?

Reply:
Yes, but note that line is used for both files and _directories_ in the code block below this...